testing/bats/doltgres.bats

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bats
+load $BATS_TEST_DIRNAME/setup/common.bash
+
+setup() {
+    setup_common
+    stash_current_dolt_user
+    unset_dolt_user
+}
+
+teardown() {
+    restore_stashed_dolt_user
+    teardown_common
+}
+
+@test 'doltgres: DOLTGRES_DATA_DIR set to current dir' {
```

Comment:
This test doesn't do what it says (you don't set DOLTGRES_DATA_DIR to any value)

Reply:
`DOLTGRES_DATA_DIR` is set in the `start_sql_server_with_args` function, but I'll make it so it gets defined in the test instead.